.travis.yml

```diff
@@ -31,10 +31,6 @@ jobs:
       name: Go vet
     - script: "docker run -v $PWD:$PWD -w $PWD quay.io/coreos/golang-testing golint -set_exit_status installer/..."
       name: Go lint
-    - script: "chmod 0777 $PWD && docker run -v $PWD:$PWD:rw -w $PWD $BAZEL_IMG bazel test terraform_fmt --test_output=all"
-      name: Terraform tests
-    - script: "chmod 0777 $PWD && docker run -v $PWD:$PWD:rw -w $PWD $BAZEL_IMG bazel test installer:cli_units --test_output=all"
-      name: Installer unit tests
```

Comment:
I'd leave these in for now to make sure we're covered until Prow picks them up (the Terraform linter is in flight with openshift/release#1124).